src/solver/conv_hip_implicit_gemm_grouped_fwd_xdlops.cpp

```diff
@@ -454,8 +454,7 @@ bool ConvHipImplicitGemmGroupFwdXdlops::IsApplicable(
     // needed because layout transpose kernel does not support non-packed tensors
     if(problem.IsLayoutDefault() && problem.HasNonPackedTensors())
         return false;
-    const std::string& arch = ctx.GetStream().GetDeviceName();
-    if(!(arch == "gfx908" || arch == "gfx90a"))
+    if(!ck_utility::is_ck_whitelist(ctx.GetStream().GetDeviceName()))
```

Comment:
@iq136boy we have some issues with this line:
```
[2024-02-19T15:17:40.948Z] /home/jenkins/workspace/MLLIBS_MIOpen_dfeng_gfwd_fix/src/solver/conv_hip_implicit_gemm_grouped_fwd_xdlops.cpp:457:9: error: use of undeclared identifier 'ck_utility'

[2024-02-19T15:17:40.948Z]     if(!ck_utility::is_ck_whitelist(ctx.GetStream().GetDeviceName()))

[2024-02-19T15:17:40.948Z]         ^
```

Reply:
@junliume let me check.